scylla/src/transport/session_test.rs

```diff
@@ -16,42 +16,68 @@ use bytes::Bytes;
 use futures::StreamExt;
 use itertools::Itertools;
 use std::collections::{BTreeMap, HashMap};
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::time::{SystemTime, UNIX_EPOCH};
 use uuid::Uuid;
 
+static UNIQUE_COUNTER: AtomicUsize = AtomicUsize::new(0);
+pub(crate) fn unique_name() -> String {
```

Comment:
This function is used across many modules and I don't see a reason why it should be defined right here. Have you considered extracting it to some new module? I think that having a module with test helpers would push us to reduce boilerplate in tests (Nearly every test starts with reading scylla uri from env, creating session and keyspace. It could be extracted to some helper function in the future.). It would also make it easier to find useful functions to use in tests (compared to using `crate::transport::session_test::unique_name()` now).

Reply:
After some reflection, I thought it would be quite convenient to have a function that sets up `session`, creates unique keyspace and issues a `use keyspace` request. Something like:
```rust
fn new_test_session() -> Session
```
We would thus avoid having to use `format!` on every query made in tests (we wouldn't have to add kespace name to each query string thanks to the `use keyspace`).

However, I think it's not in this PR scope.

Reply:
Yes, it definitely makes sense to provide some utils like caching_session's new_for_test. I'm leaving it for a follow-up though